types/recharts/index.d.ts

```diff
@@ -1084,3 +1084,9 @@ export interface SymbolsProps extends Partial<PresentationAttributes> {
 }
 
 export class Symbols extends React.Component<SymbolsProps> { }
+
+export interface CustomizedProps {
+    component: ContentRenderer<any> | React.ReactElement;
```

Comment:
I am wondering about this `any`. The function will receive `props` and `state` as parameters. Is there a way we can enforce that? We already have all the props defined for each type of chart

Reply:
Then props can be LineChartProps, BarChartProps, ComposedChartProps, AreaChartProps, PieChartProps, RadarChartProps, RadialBarChartProps, ScatterChartProps or FunnelChartProps. The same is for the state. Also, we have no definitions for these state types. Do you really want me to create all types for the state?

Reply:
well, I guess there's nothing we can do for the state part 😞

Reply:
I don't think that these combined props without the state will be more useful than any type.

Reply:
oh yes, I agree. Sorry if it looked that you should do the Props part, I did not mean that. It doesn't make sense without the state